package.json

```diff
@@ -47,14 +62,20 @@
     "babel-plugin-module-resolver": "^2.7.1",
     "babel-preset-flow": "^6.23.0",
     "babel-preset-react-native": "1.9.1",
-    "eslint": "^3.15.0",
-    "eslint-config-airbnb": "^14.1.0",
+    "eslint": "^3.19.0",
+    "eslint-config-airbnb": "~15.0.1",
+    "eslint-config-airbnb-base": "~11.2.0",
+    "eslint-import-resolver-babel-module": "^3.0.0",
+    "eslint-plugin-flowtype": "^2.35.0",
     "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^4.0.0",
-    "eslint-plugin-react": "^6.10.0",
-    "eslint-plugin-react-native": "^2.2.1",
+    "eslint-plugin-jsx-a11y": "^5.0.1",
+    "eslint-plugin-react": "^7.1.0",
```

Comment:
That's a good question. I think fixed versions doesn't really matter since we're using yarn. Maybe it's an Atom bug?